build.sh

```diff
@@ -14,4 +14,4 @@ else
     die
 fi
 
-(cd src && ./build.sh) || die
+(cd src && ./build.sh "$@") || die
```

Comment:
drop this, I am picking it for the pure-bootstrap branch; also, no quotes, it could be multiple arguments.

Reply:
Nah, "$@" is precisely the thing that works with multiple arguments with spaces.